File: Eclipse/Fumigation/src/primenumber.java
import java.util.Scanner;

public class primenumber {
	
	public static void main(String[] args) {
		int c = 0;
		Scanner s=new Scanner(System.in);
		System.out.println("Enter a number:");
		int num=s.nextInt();
		for(int i=2;i<=num/2;i++)
		{	c=0;
			if(num%i==0)
			{
				c=1;
				break;
			}
		}
		if(c==0 && num!=1 && num!=0)
		{
			System.out.println("Prime number");
		}
		else
		{
			System.out.println("not prime number");
		}
		
		
	}
}

File: Pattern_array_practice/Practice c/PrimeNumber.c
#include <stdio.h>
#include <stdlib.h>

int main(void){
    int num,i;
    int flag=0;
    printf("enter a number");
    scanf("%d",&num);

    for(i=2;i<=num/2;i++){
       if( num%i==0){
        flag=1;
        break;
       }          

    }
    
    if(flag==0){
        printf("prime number");
    }else{
        printf("not prime");
        }
}